packages/expo-screen-capture/ios/ScreenCaptureModule.swift

```diff
@@ -69,4 +98,48 @@ public final class ScreenCaptureModule: Module {
       "body": nil
     ])
   }
+
+  private func preventScreenshots() {
+    guard #available(iOS 13.0, *) else {
+      return
+    }
```

Comment:
This shouldn't be necessary as we [don't support anything below iOS 15.1](https://docs.expo.dev/versions/latest/#support-for-android-and-ios-versions)
```suggestion
```